proposals/README.md

```diff
@@ -0,0 +1,42 @@
+# Proposing Changes to Cadence
+
+## Introduction
+
+The design process for changes to Cadence is modeled on the [proposal process used by the M3 project](https://github.com/m3db/proposal).
+
+## Process
+
+- [Create an issue](https://github.com/uber/cadence/issues/new) describing the proposal.
+
+- Like any GitHub issue, a proposal issue is followed by an initial discussion about the suggestion. For Proposal issues:
+
+  - The goal of the initial discussion is to reach agreement on the next step: (1) accept, (2) decline, or (3) ask for a design doc.
+  - The discussion is expected to be resolved in a timely manner.
+  - If the author wants to write a design doc, then they can write one.
+  - A lack of agreement means the author should write a design doc.
+
+- If a Proposal issue leads to a design doc:
+
+  - The design doc should be presented as a Google Doc and must follow [the template](https://docs.google.com/document/d/1hpWpy5MB5l8uXfnl23lebhx-dvo79vb1jFUOyAtIHJw/edit?usp=sharing).
+  - The design doc should be linked from the opened GitHub issue.
+  - The design doc should only allow edit access to authors of the document.
+  - Do not create the document from a corporate G Suite account, if you wish to edit from a corporate G Suite account then first create the document from a personal Google account and grant edit access to your corporate G Suite account.
+  - Comment access should be granted explicitly to [cadence-discussion@googlegroups.com](mailto:cadence-discussion@googlegroups.com) so that users viewing the document display under their account names rather than anonymous users.
+  - Comment access should also be accessible by the public, make sure this is the case by clicking "Share" and "Get shareable link" ensuring to select "Anyone with the link can comment".
```

Comment:
public, make sure -> public. Make sure
ensuring to select -> ensuring you select